### ISSUE TITLE
Color Contrast of Equal Signs on Bar Chart View?

### ISSUE THREAD

**Issue by kathy-phet** [phetsims/balancing-chemical-equations#210]
@Nancy-Salpepi - Can you check color contrast for yellow to light blue background when balanced. Might need a thicker black stroke around the equal signs in the bar chart view (something similar to the stroke on the arrow, it’s currently quite thin). 

![Image](https://github.com/user-attachments/assets/b04a1cb8-9df0-4dec-b183-dfd5cd8f81d9)

**Comment by Nancy-Salpepi** [comment]
Thanks @kathy-phet. 
Cathy Carter helped me test this and it does indeed fail. 

@pixelzoom is it possible to make the yellow equals signs a bit larger and increase the black stroke thickness around it? We thought that if we kept the size as is, the larger stroke would drown out the yellow. Should we also increase the size of ≠ so that they are the same?

**Comment by pixelzoom** [comment]
Nancy and I worked on this synchronously, summary of changes:

* Increased font size of equality operator (=,≠) from 80 to 100.
* Increased lineWidth from 1 to 1.5 for equality operators, and right arrow in equations.
* Changed stoke for smile/frown face from mustard to black (lineWidth is still 1).

**Comment by pixelzoom** [comment]
Nancy and I worked on this synchronously.  As requested, we added the thicker stroke and increase the size of the equality operators.  We also identified several other places when a thicker stroke is needed for similar reasons.
 
summary of changes:

* Increased font size of equality operator (=,≠) from 80 to 100.
* Increased lineWidth from 1 to 1.5 for equality operators, and right arrow in equations.
* Increased stroke for smile/frown face from mustard to black (lineWidth is still 1).
* Increased stroke on balance beams from 1 to 1.25.

**Comment by pixelzoom** [comment]
@kathy-phet want to take a look?  Close if OK.

**Comment by pixelzoom** [comment]
Reviewed with @kathy-phet on Zoom, looks good, closing.
